### PR TITLE
feat: [SRM-14325]: removed feature flag SRM_INTERNAL_CHANGE_SOURCE_FF

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesSourceCard/ChangesSourceCard.utils.ts [hunk 1]
```diff
@@ -35,29 +35,13 @@ const labelByCategory = (
...
-        ? 0
-        : category?.countInPrecedingWindow || 0
-    const categoryPercentage: number = ((count - previousCount) / previousCount) * 100
-    return {
-      count,
-      id: categoryType ?? '',
-      label: labelByCategory(categoryType, getString),
-      percentage: isNaN(categoryPercentage) ? 0 : categoryPercentage === Infinity ? 100 : categoryPercentage
-    }
+  return {
```
Comment (karansaraswat19) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesSourceCard/ChangesSourceCard.utils.ts:40]: here also

File: src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx [hunk 2]
```diff
@@ -206,33 +203,12 @@ export default function ChangeTimeline(props: ChangeTimelineProps): JSX.Element
...
-                )
-              },
-              {
-                labelName: getString('cv.changeSource.tooltip.incidents'),
-                data: createTimelineSeriesData(ChangeSourceTypes.Alert, getString, categoryTimeline?.Alert)
-              }
-            ]
-      }
+      timelineRows={Object.entries(categoryTimeline || defaultCategoryTimeline).map(timeline => {
+        return {
```
Comment (karansaraswat19) [inline src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx:207]: I have mistakenly put return in arrow functions. We can remove them.

File: src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.utils.ts [hunk 3]
```diff
@@ -137,51 +137,22 @@ export const labelByCategory = (categoryType: string, getString: UseStringsRetur
...
-        },
-        {
-          key: ChangeSourceTypes.Infrastructure,
-          count: sumBy(filterInfra, 'count'),
-          message: createTooltipLabel(sumBy(filterInfra, 'count'), ChangeSourceTypes.Infrastructure, getString)
-        }
-      ]
-    }
+    return Object.entries(categoryTimeline).map(category => {
+      return {
```
Comment (karansaraswat19) [inline src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.utils.ts:146]: here too

**Review by karansaraswat19 — CHANGES_REQUESTED** [review]

File: src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx (hunk 2 above)
Comment (deepesh-ui) [reply src/modules/85-cv/components/ChangeTimeline/ChangeTimeline.tsx:207]: done

**Review by praneshg239 — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**